### PR TITLE
adding_documentation_to_DP_rheology

### PR DESCRIPTION
**Pull request by cedrict**
Pull Request Checklist. Please read and check each box with an X. Delete any part not applicable. Ask on the [forum](https://community.geodynamics.org/c/aspect) if you need help with any step.

*Describe what you did in this PR and why you did it.*

### Before your first pull request:

* [ x] I have read the guidelines in our [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md) document.

### For all pull requests:

* [x ] I have followed the [instructions for indenting my code](../blob/main/CONTRIBUTING.md#making-aspect-better).

### For new features/models or changes of existing features:

* [ ] I have tested my new feature locally to ensure it is correct.
* [ ] I have [created a testcase](https://aspect-documentation.readthedocs.io/en/latest/user/extending/testing/writing-tests.html) for the new feature/benchmark in the [tests/](../blob/main/tests/) directory.
* [ ] I have added a changelog entry in the [doc/modules/changes](../blob/main/doc/modules/changes) directory that will inform other users of my change.

### CODE REVIEW

File: source/material_model/rheology/drucker_prager.cc [hunk 1]
```diff
@@ -72,6 +72,14 @@ namespace aspect
       {
         const double sin_phi = std::sin(angle_internal_friction);
         const double cos_phi = std::cos(angle_internal_friction);
+
+        // The expression below for differs from Eq. 9 of Glerum et al, 2018.
+        // There are actually three different ways of chosing this parameter, which
```
Comment (naliboff) [inline source/material_model/rheology/drucker_prager.cc:77]:
```suggestion
        // There are actually three different ways of choosing this parameter, which
```

**Review by naliboff — CHANGES_REQUESTED** [review]
@cedrict - Thanks for adding this documentation! One minor spelling correction needed (I suggest squashing the commits), and then good to merge.

**Comment by cedrict** [comment]
@naliboff done

**Review by gassmoeller — APPROVED** [review]
Just a small fix, otherwise this is good to go. @naliboff: do you agree?

**Review by naliboff — APPROVED** [review]

**Pull request merged**